docs/contributing/setup-windows.md

```diff
@@ -74,16 +74,19 @@ Open a shell and run this command:
 $ python --version
 ```
 
-If you see the output `Python 2.7.x`, you're good to go!
+If you see the output `Python 3.9.x`, you're good to go!
 
 If you see an error about being unable to find `python`, that probably means you
-don't have Python installed. You can install Python 2.7 from the
+don't have Python installed. You can install Python 3.9 from the
 [Python website](https://www.python.org/downloads/windows/).
 
 **Python installation notes:**
 
- - Let Python install into the default suggested path (`c:\Python27`), otherwise
+ - Let Python install into the default suggested path (`c:\Python39`), otherwise
    you'll have to configure `node-gyp` manually to look at a different path.
+ - In the case you already have or need to have Python on a different path, set the `PYTHON` environment variable or the npm's `python` config key to your Python's executable path:
```

Comment:
Minor wording changes:
```suggestion
 - In case you already have or need to have Python on a different path, set the `PYTHON` environment variable or npm's `python` config key to your Python's executable path:
```

Reply:
Thank you! 💛 I'm aware that I'm overusing articles, probably in a hope that at least some of them are actually placed correctly, since I have never properly learned the rules how to use them. 😊 Disclaimer: English is not my mother tongue.

Off topic: I would appreciate if you can suggest a good resource to learn how to use the English articles! 🙂